Oqtane.Client/Modules/Admin/Modules/Settings.razor

```diff
@@ -162,14 +162,7 @@
         module.Permissions = _permissionGrid.GetPermissions();
         await ModuleService.UpdateModuleAsync(module);
 
-        if (_settingsModuleType != null)
-        {
-            var moduleType = Type.GetType(ModuleState.ModuleType);
-            if (moduleType != null)
-            {
-                moduleType.GetMethod("UpdateSettings")?.Invoke(_settings, null); // method must be public in settings component
-            }
-        }
+        _settings?.GetType().GetMethod("UpdateSettings")?.Invoke(_settings, null); // method must be public in settings component
```

Comment:
We have to introduce interface for settings form (ISettingForm or ISettingControl),
I have now created one setup form and have had trouble remembering how it works.